### DISCUSSION TITLE
Welcome to nuxt-snackbar Discussions!

### DISCUSSION THREAD

**Discussion by modbender** [modbender/nuxt-snackbar#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GreyXor** [comment]
Hello, my idea is to make it working even in a plugin, like with vue3-toastify.
https://github.com/modbender/nuxt-snackbar/issues/19#issuecomment-2200547995

**Reply by modbender** [reply]
nuxt-toastify coming soon

**Reply by GreyXor** [reply]
Great news! Thanks
So nuxt-toastify will work in plugins? but not nuxt-snackbar?

**Reply by modbender** [reply]
I've yet to see it's working so I'm not sure

**Reply by modbender** [reply]
@GreyXor please try out https://github.com/modbender/nuxt-toastify
The workings of the toastify is indeed very different from snackbar and might allow you to use it within a plugin, I will test the same later on.

**Reply by GreyXor** [reply]
Thanks! I'll continue the discussion there. If nuxt-toastify can be used in a plugin, I still believe it's much more practical for our Nuxt applications.
